safe/test/utilities.py

```diff
@@ -100,6 +117,29 @@ def get_qgis_app():
         # noinspection PyCallByClass,PyArgumentList
         QCoreApplication.setApplicationName('QGIS2InaSAFETesting')
 
+        # Save some settings
+        if qsetting:
+            settings = QSettings(qsetting)
+        else:
+            settings = QSettings()
+
+        settings.setValue('locale/overrideFlag', True)
```

Comment:
Use our `set_general_setting` method

Reply:
yes I changed it again for testing reason, I'm aware of it